torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -177,12 +177,14 @@ class SubclassCreationMeta:
     # both of its inner elements are TwoTensors, then the
     # arg_count of the outer-most sublass will be 4
     arg_count: int
+    # Mark where or not symints were included
+    included_subclass_symints: bool
```

Comment:
nit: I think this  bool is only actually used for assertion checking purposes (is that right?) maybe mention that in the comment?